src/components/Asset/Edit/FormEditMetadata.tsx

```diff
@@ -103,6 +103,7 @@ export default function FormEditMetadata({
                   ? timeoutOptionsArray
                   : field.options
               }
+              prominentHelp={field.name === 'links'}
```

Comment:
should we apply the same mechanism on the edit form?

Reply:
would say yes. But looking at mechanism, conceptionally, it should be up to content author to decide about tooltip help or prominent help on the respective field content, so would put the `prominentHelp` key in the content file and work from there. 

Because otherwise we always would have to edit specific components when we decide to use prominent help on another field too. Once we get this key from the content file, we don't need to think about adding this as a prop to some components or not